blurring_as_a_service/inference_pipeline/submit_inference_pipeline.py

```diff
@@ -10,58 +12,90 @@
 
 
 @pipeline()
-def inference_pipeline(mounted_root_folder, relative_paths_files_to_blur, model):
-    outputs = BlurringAsAServiceSettings.get_settings()["inference_pipeline"]["outputs"]
+def inference_pipeline():
```

Comment:
Not the best practice to instantiate global objects, this is why I was passing everything as params.

Reply:
https://www.baeldung.com/cs/global-variables